geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -739,6 +736,7 @@ public EnumInfo getEnumById(int id) {
    */
   private void updateClassToTypeMap(PdxType type) {
```

Comment:
The method needs to be changed; as we are updating typeToId map.

Reply:
done